src/Manifolds.jl

```diff
@@ -367,6 +367,9 @@ include("manifolds/Unitary.jl")
 include("manifolds/Rotations.jl")
 include("manifolds/Orthogonal.jl")
 
+# shape spaces require Sphere
+include("manifolds/ShapeSpace.jl")
```

Comment:
Maybe also call the file `KendallShapeSpaces.jl`? Or we could also split the two manifolds into two files; they can still share one documentation page but it would also make it easier to have two sections for the two manifolds in there.

Reply:
It was easier to put it in one file for development but I can split it now.

Reply:
I've split the file in the last commit.